packages/terser/src/index.js

```diff
@@ -117,6 +148,7 @@ function terserDirectory(input) {
 }
 
 if (!inputs.find(isDirectory) && inputs.length) {
+  process.argv = processSourceMapArg(process.argv, output);
```

Comment:
why mess with the argv if we are just a pass-through to terser? it should be a no-op

Reply:
terser reads the args back out of `process.argv` it's the only api we have for it

Reply:
yeah I know, but why do you need to do any processing on argv?

Reply:
we need to process the source map argument, in processing we modify it to include the extra option, the same as the directory method

Reply:
Sorry I still don't get it. If you're not doing a directory, you don't need our wrapper. The original terser arguments should work as-is

Reply:
>The original terser arguments should work as-is

The original terser arguments don't have the `url` set in the sourcemap options, calling this methods sets it.

Reply:
I see, so then what I'm looking for is that the original terser arguments should include `url` in the sourcemap options so that the original binary works in the single-file case.

Reply:
hmm ok ill see what I can do, but I assume this means we'll be duplicating the logic in the function I added between here and in the `.bzl` that goes with this.

Reply:
Turns out no logic is required in the .bzl, we know the exact name.